### ISSUE TITLE
feat requ: when no heads exits, disable all header based features please

### ISSUE THREAD

**Issue by mckaygerhard** [Mobius1/Vanilla-DataTables#5]
i use the plugin and disable all options to only render a very basic table..

OF COURSE THE TABLE NO HAS HEADER so got this error:

```
Uncaught TypeError: Cannot read property 'rows' of null vanilla-dataTables.js:250
build vanilla-dataTables.js:250
Plugin vanilla-dataTables.js:601
(anonymous function)
```
HAPENED DUE THE PAGINATION I GUESS, based on heading NOTE in documentation, so a header - less can be do, only disabling all heading based features, this help to solve the issue #4 and make it work with normal tables tags

the code init to do was:

```
</table><script src="http://127.0.0.1/~general/codeigniter/appweb/scripts/vanilla-dataTables.js"></script>
<script>
var optionstable306 = { sortable: false, searchable: false, fixedcolumn: false, perPage: 50 };
var datatable306 = new DataTable(document.getElementById('table306'), optionstable306);</script>
```

i set some hacks on the js code but i'll parse and post feedback here after some tests..

**Comment by mckaygerhard** [comment]
@Mobius1 i respond here due the facts:

i'm still studin if a rendering with no headers its possible, based on #5 seems that its possible.. 
this its the rendering results (with some errors): its xtremedly basic and u will think "well if no headers and will paint very basic, so for what use datatables?"...

well its due i'm integrating into codeigniter modified version, so if the table open tag does not providing header, will render very baic.. 
in the other hand, well if script initialization was not detect headers, event luach a exception u could launch a console log or only wrning and render only basic.. so developer got less frustrating (yeah i spend some hours to got it)

![rendering-withoutheaders](https://cloud.githubusercontent.com/assets/1571724/24075220/930af7ba-0bed-11e7-9f0e-e59b254cffce.png)

**Comment by mckaygerhard** [comment]
i got a decent code, tested i propose:

around line 560

```
		if (table.tHead === null)
		{
			_.options.sortable = false;
			_.options.searchable = false;
			_.options.fixedColumns = false;
		}
```

**does all the trick and solves both problems.. but in the other hand i wish a miracle, ...could be possible make search without headers?**

**Comment by Mobius1** [comment]
This is fixed in the [latest release](https://github.com/Mobius1/Vanilla-DataTables/releases/tag/1.0.4). :)

**Comment by mckaygerhard** [comment]
hey u made a totally different way of, the commit 313ec92 does not introduced new regresions?

ill' closes the #5 and #4 when i tested in good way, give a few hours, and please take in consideration the isse about got the pagination on the header

**Comment by mckaygerhard** [comment]
AWESOME; i'm able to search without headers, so with headers too!

closes #5 and #4 with commit 313ec92 THANKS!
